src/Makefile.am

```diff
@@ -174,6 +177,7 @@ noinst_HEADERS = \
 	db_sub_source.hh \
 	doc_status_source.hh \
 	doctest.hh \
+	$(doctest_vendored_h) \
```

Comment:
My goal with this was to avoid building the bundled doctest when we are using the system one. If you know a better way I'm all ears!